package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "medic-smsparser",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "underscore": "^1.3.3"
+  },
+  "devDependencies": {
+    "nodeunit": "^0.11.0",
+    "sinon": "^2.1.0"
+  },
+  "scripts": {
+    "test": "./node_modules/.bin/nodeunit tests/unit"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
```

Comment:
Shouldn't this be "Apache-2.0" same as webapp? Don't want to have to get the lawyers involved...